frontend/src/components/managePermissions/table/DataTableHeader.vue

```diff
@@ -64,7 +64,7 @@ const tableHeaderCustomText = computed(() => {
         </span>
     </div>
 
-    <div class="search-container">
+    <div class="add-search-container">
```

Comment:
yeah I feel this naming is still weird... maybe call it "utility-container" ?

Reply:
I like your suggestion. I'll replace it, thank you!